### PR TITLE
Add F-DDrace entities

### PR DESCRIPTION
**Pull request by fokkonaut**
## Checklist

- [x] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Comment by def-** [comment]
Why are different entities required? Isn't it a subset of DDNet entities?

**Comment by fokkonaut** [comment]
> Why are different entities required? Isn't it a subset of DDNet entities?

It has a lot of extra tiles so I thought it would be useful to have it in DDNet client aswell, since it's in my 0.7 client

File: src/game/client/gameclient.cpp [hunk 1]
```diff
@@ -1064,6 +1065,7 @@ static CGameInfo GetGameInfo(const CNetObj_GameInfoEx *pInfoEx, int InfoExSize,
 		City = IsCity(pFallbackServerInfo);
 		Vanilla = IsVanilla(pFallbackServerInfo);
 		Plus = IsPlus(pFallbackServerInfo);
+		FDDrace = IsFDDrace(pFallbackServerInfo);
```
Comment (heinrich5991) [inline src/game/client/gameclient.cpp:1068]: This fallback and the `IsFDDRace` function isn't needed. Just use `FDDRace = false` here.

**Comment by heinrich5991** [comment]
LGTM, but there are still errors in the CMakeLists.txt, you need to list the new data file.

**Comment by heinrich5991** [comment]
Could you squash the commits (`git rebase -i HEAD^^^` and then mark the second and third commit as `squash`) so we don't end up with both images in the repository (increases size).

**Comment by heinrich5991** [comment]
Awesome, thanks. :)

bors r+

**Pull request merged**